vscode/microsoft-kiota/src/utilities/status-bar.ts

```diff
@@ -0,0 +1,25 @@
+import * as vscode from "vscode";
+
+import { ExtensionContext, StatusBarItem } from "vscode";
+import { getKiotaVersion } from "./getKiotaVersion";
+import { kiotaOutputChannel } from "./logging";
+
+async function updateStatusBarItem(context: ExtensionContext, kiotaStatusBarItem: StatusBarItem): Promise<void> {
+  try {
+    const version = await getKiotaVersion(context, kiotaOutputChannel);
+    if (!version) {
```

Comment:
Can we improve this, even in a separate issue?

Reply:
No changes to the logic